core/src/term/mod.rs

```diff
@@ -2721,7 +2721,7 @@ pub mod make {
         Term::OpN(op, args.into_iter().map(T::into).collect()).into()
     }
 
-    pub fn apply_contract<T>(
+    pub fn contract_apply<T>(
```

Comment:
Nitpick, but I think prefer the old name. The primop has the name `contract/apply`, which is a bit like a poor man's namespacing `contract::apply`. To avoid confusion, the corresponding variants `UnaryOp`, `BinaryOp`, and `NAryOp` have the exact same name (replacing the `/` by a separation in camel case). But I feel like we don't need to do that for such a standalone helper function.